src/seedu/addressbook/data/person/Name.java

```diff
@@ -45,6 +45,11 @@ public static boolean isValidName(String test) {
     }
 
     @Override
+    public String getPrintableString() {
+        return "Name: " + fullName + "\n";
```

Comment:
This `\n` is a bit problematic because this prevents from concatenating printable strings into a single line.

Reply:
My intention was to print it into a few lines to increase readability, instead of concatenating everything into a single line.